### PR TITLE
Fixes Travis Docs Deploy

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Fixes the docs deploy stage in Travis. Just needed a little directory navigation TLC.

### What design trade-offs/decisions were made?

Added a bunch of `cd`'s that I bet I wouldn't need if I knew `node` a little better.

### How do I test this PR?

Push to master :) Sike, the stage is set to only happen on a push to master so you can't really test without ssh'ing into Travis and running the steps yourself.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [#655]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
@vinnyjth there's no Travis link here haha, not sure how to trigger the status check.

**Comment by vinnyjth** [comment]
@redreceipt 
I managed to find this in the requests tab of Travis.

`Could not parse ApollosProject/apollos-prototype/.travis.yml@06393c8b6529280f`

**Comment by redreceipt** [comment]
🙄 missing a dash...

File: .travis.yml [hunk 1]
```diff
@@ -45,22 +45,23 @@ jobs:
...
+         - cd packages/apollos-ui-kit
+         - cat ../../docs/website/sidebars.json | packages/apollos-ui-kit/docs/updateSidebar.sh
+         - cd ../../
+         - cp -f packages/apollos-ui-kit/docs/generated/sidebars.json docs/website/sidebars.json
+         # push new docs
+         - cd docs
+         - git remote
+         - git config user.email ${GH_EMAIL}
+         - git config user.name ${GH_USER}
+           git add .
```
Comment (redreceipt) [inline .travis.yml]:
```suggestion
         - git add .
```

File: .travis.yml [hunk 2]
```diff
@@ -45,22 +45,23 @@ jobs:
...
-    #     - git config user.name ${GH_USER}
-    #       git add .
-    #     - git commit -am ${MESSAGE}
-    #     - git push "https://${GH_TOKEN}@${GH_DOCS_REPO}" master > /dev/null 2>&1
+     - stage: Deploy Docs
+       script:
+         - yarn generate-docs
+         - MESSAGE=$(git log --format=%B -n 1 $TRAVIS_COMMIT)
+         - git clone git://${GH_DOCS_REPO}
+         # copy over ui-kit docs
```
Comment (redreceipt) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 3]
```diff
@@ -45,22 +45,23 @@ jobs:
...
+         - cp -fr packages/apollos-ui-kit/docs/generated/* docs/docs/ui-kit
+         # update sidebar
```
Comment (redreceipt) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 4]
```diff
@@ -45,22 +45,23 @@ jobs:
...
+         - cd packages/apollos-ui-kit
+         - cat ../../docs/website/sidebars.json | packages/apollos-ui-kit/docs/updateSidebar.sh
+         - cd ../../
+         - cp -f packages/apollos-ui-kit/docs/generated/sidebars.json docs/website/sidebars.json
+         # push new docs
```
Comment (redreceipt) [inline .travis.yml]:
```suggestion
```

**Comment by redreceipt** [comment]
I thought I fixed it but it won't trigger a build

**Comment by vinnyjth** [comment]
@redreceipt I'm seeing the same error in travis.
https://travis-ci.org/ApollosProject/apollos-prototype/requests

**Comment by redreceipt** [comment]
I didn't uncomment the lines right...triggered now

**Comment by redreceipt** [comment]
finally broke through, now I'm going to ssh in and test the script before merging

**Pull request merged**